src/features/Checkin/Main.tsx

```diff
@@ -114,9 +114,12 @@ class CheckinContainer extends React.Component<{}, ICheckinState> {
           }
         );
       }
+      /*
+      Removed as shirt size has been removed from account
```

Comment:
We need to know what shirt size the hacker is when they check in. Make sure that this information is still accessible.

Reply:
ah! was wondering why it was in profile instead of application. I think we'll take `shirtSize` from application and output to check-in screen b/c we don't want people to change their shirt size after applications close and deciding during account creation might be too early to make view-only. what do you think?

Reply:
Sure. We thought to make it part of profile so that all users can enter in their short data, since we wanted shirts for everyone